doc/src/manual/constructors.md

```diff
@@ -573,6 +574,7 @@ Closest candidates are:
 ```
 
 This constructor will be invoked by the syntax `SummedArray(a)`. The syntax `new{T,S}` allows
-specifying parameters for the type to be constructed, i.e. this call will return a `SummedArray{T,S}`.
-`new{T,S}` can be used in any constructor definition, but for convenience the parameters
-to `new{}` are automatically derived from the type being constructed when possible.
+specifying parameters for the type to be constructed and this call will return a
+`SummedArray{T,S}`. `new{T,S}` can be used in any constructor definition (recall, the parameters to
```

Comment:
Perhaps for consistency with the near-identical statements above, something along the lines of the following?

"`new{T,S}` can be used in any constructor definition. For convenience, the type parameters to `new{}` are automatically derived from the type being constructed when possible; in the example above, however, the type parameters must be specified explicitly."